consensus/polybft/consensus_runtime.go

```diff
@@ -1031,9 +1031,12 @@ func (c *consensusRuntime) IsValidSender(msg *proto.Message) bool {
 	c.lock.RLock()
 	defer c.lock.RUnlock()
 
-	err := c.fsm.ValidateSender(msg)
-	if err != nil {
-		c.logger.Error("invalid sender", "error", err)
+	if c.fsm == nil {
+		return false
```

Comment:
Could you add a log there?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1054/commits/66ced16002fd71851da727dd3a734e9ef76a02b4